addon/templates/components/frost-radio-button.hbs

```diff
@@ -0,0 +1,9 @@
+{{#radio-button
+    radioId=radioId
```

Comment:
radio button is the component that this uses. radioId is a property of that, I wouldn't be able to change this unfortunately